test/run

```diff
@@ -28,23 +28,23 @@ run_doc_test
 THISDIR=$(dirname ${BASH_SOURCE[0]})
 source "${THISDIR}"/test-lib.sh
 
-check_result() {
-  local result="$1"
-  if [[ "$result" != "0" ]]; then
-    TESTCASE_RESULT=1
-  fi
-  return $result
-}
-
-function get_cid() {
-  local id="$1" ; shift || return 1
-  echo $(cat "$CID_FILE_DIR/$id")
-}
-
-function get_container_ip() {
-  local id="$1" ; shift
-  docker inspect --format='{{.NetworkSettings.IPAddress}}' $(get_cid "$id")
-}
+	check_result() {
+	  local result="$1"
+	  if [[ "$result" != "0" ]]; then
+	    TESTCASE_RESULT=1
+	  fi
+	  return $result
+	}
+
+	function get_cid() {
+	  local id="$1" ; shift || return 1
+	  echo $(cat "$CID_FILE_DIR/$id")
+	}
+
+	function get_container_ip() {
```

Comment:
Can be removed in favor of: https://github.com/sclorg/container-common-scripts/blob/646af729971641b5f6ec9de977ec13a563b75bbc/test-lib.sh#L344